main.cpp

```diff
@@ -1908,33 +1930,84 @@ void verify_command::execute(device_map &devices) {
     }
 }
 
+static int reboot_device(libusb_device *device, bool bootsel, uint disable_mask=0) {
+    // ok, the device isn't in USB boot mode, let's try to reboot via baud rate hack
```

Comment:
Is this comment correct? Should it say "reboot via reset vendor interface" instead?
Should this PR _also_ include the baud-rate-hack, or will that be added in a later PR?

Reply:
it should say that; resetting by baud is not supported, as it can't be done via libusb when the device is claimed by a comms driver, and finding the comms device is complex & platform dependent